source/partials/release_notes/_release-19-4-0.md.erb

```diff
@@ -14,7 +14,7 @@ Users can also utilize the newly introduced `branch` attribute on Mercurial mate
 
 <h4>Introduced support for Java 12</h4>
 
-Starting this release, GoCD has now added support for Java 12. All the docker images and installers for Windows and OSX are bundled with the supported Java versions.
+Starting this release, GoCD has now added support for Java 12. All the docker images and installers for Windows and OSX are bundled with a supported version of Java (not necessarily 12).
```

Comment:
> (not necessarily 12)

We bundled Java 12 for this release: https://github.com/gocd/gocd/commit/1e344e8848ceda4ce67cc202813cdb4f95698e45

Reply:
not for docker alpine gocd

Reply:
Sry, missed that.